ui/src/app/components/runtime-configuration-panel/start-stop-environment-button.tsx

```diff
@@ -42,7 +42,7 @@ export const StartStopEnvironmentButton = ({
     dataTestId,
     styleOverrides = {},
     onClick = null,
-  } = switchCase(
+  } = switchCase<UserEnvironmentStatus, any>(
```

Comment:
Not sure how to type this, but since it is assigned right above, maybe it's fine as any.